src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
```

Comment:
You are right @ahocevar, we no longer listen for the global layer change event.
So all features are kept and it makes sense to try to update each object opacity.
The update code should be moved out of the synchronizer and placed in a core function.